Web_Development/FrontEnd/Html_Table_Tags/Html_Tables.md

```diff
@@ -0,0 +1,175 @@
+
+## **HTML_Tables**
```

Comment:
@VaishnaviPhadtare Reduce the plagiarism, try framing the plagiarized bits of the content in your own words.